File: QE-selenium python/Activity19.py
from selenium import webdriver
import time

driver = webdriver.Firefox()

driver.get("https://training-support.net/webelements/alerts")

print(driver.title)

driver.execute_script("document.getElementById('confirm').scrollIntoView(true);")
driver.execute_script("document.getElementById('confirm').click();")

time.sleep(1)
alert = driver.switch_to.alert
print(alert.text)
alert.accept()

driver.execute_script("document.getElementById('confirm').click();")

time.sleep(1)
alert = driver.switch_to.alert
alert.dismiss()

driver.quit()

File: QE selenium java/Activity19.java
package demos;
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity19 {
    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/webelements/alerts");

        System.out.println(driver.getTitle());

        driver.findElement(By.id("confirm")).click();

        Alert alert = driver.switchTo().alert();
        System.out.println(alert.getText());
        alert.accept();

        driver.findElement(By.id("confirm")).click();

        alert = driver.switchTo().alert();
        alert.dismiss();

        driver.quit();
    }
}

